lib/internal/Magento/Framework/App/Utility/Files.php

```diff
@@ -938,7 +938,7 @@ private function accumulateThemeStaticFiles($area, $locale, $filePattern, &$resu
                     $themePath . "/*_*/web/i18n/{$locale}"
                 ];
                 $this->_accumulateFilesByPatterns($paths, $filePattern, $files);
-                $regex = '#^' . $themePath .
+                $regex = '#^' . preg_quote($themePath) .
```

Comment:
You'll also need to provide the second parameter to `preg_quote` to make sure the chose delimiter `#` gets escaped, too (so that the escape is always 100% correct).

Reply:
Yep, right. Thanks @oliverklee